src/main/java/co/cask/hydrator/plugin/batchSource/KafkaBatchSource.java

```diff
@@ -304,7 +304,10 @@ public Schema getMessageSchema() {
     }
 
     public void validate() {
-      getBrokerMap();
```

Comment:
Yes. The conflict exists no matter how we deal with the table name, since anyone can create table with same name in other places. We currently do not have a good way to deal with it.